### ISSUE TITLE
Cannot find module 'form-data'

### ISSUE THREAD

**Issue by alanshaw** [ipfs/js-ipfs#2482]
In 0.38.0-rc.5:

```console
$ iim use js
✔ selected js-ipfs version 0.38.0-rc.5
✔ installed js-ipfs 0.38.0-rc.5
✔ installed binary at /Users/alan/.iim/dists/js-ipfs@0.38.0-rc.5/ipfs
✔ initialized IPFS at /Users/alan/.iim/js-ipfs@0.38.0-rc.5
✔ configured IPFS
✔ symlinked /Users/alan/.iim/dists/js-ipfs@0.38.0-rc.5/ipfs -> /Users/alan/.iim/dists/current
✔ symlinked /Users/alan/.iim/dists/js-ipfs@0.38.0-rc.5/ipfs -> /usr/local/bin/ipfs
ℹ using repo at /Users/alan/.iim/js-ipfs@0.38.0-rc.5
🚀 IPFS is ready to use

$ ipfs daemon &
ipfs daemon &
[1] 81191

$ Initializing IPFS daemon...
js-ipfs version: 0.38.0-rc.5
System version: x64/darwin
Node.js version: 12.10.0
Swarm listening on /ip4/0.0.0.0/tcp/4001/ipfs/Qma1P1VG7ueSbec4bGcNb13HUCrSDNHd3ytDXG47GnF5GU
Swarm listening on /ip6/::/tcp/4001/ipfs/Qma1P1VG7ueSbec4bGcNb13HUCrSDNHd3ytDXG47GnF5GU
Swarm listening on /ip4/127.0.0.1/tcp/4003/ws/ipfs/Qma1P1VG7ueSbec4bGcNb13HUCrSDNHd3ytDXG47GnF5GU
Swarm listening on /p2p-circuit/ipfs/Qma1P1VG7ueSbec4bGcNb13HUCrSDNHd3ytDXG47GnF5GU
Swarm listening on /p2p-circuit/ip4/0.0.0.0/tcp/4001/ipfs/Qma1P1VG7ueSbec4bGcNb13HUCrSDNHd3ytDXG47GnF5GU
Swarm listening on /p2p-circuit/ip6/::/tcp/4001/ipfs/Qma1P1VG7ueSbec4bGcNb13HUCrSDNHd3ytDXG47GnF5GU
Swarm listening on /p2p-circuit/ip4/127.0.0.1/tcp/4003/ws/ipfs/Qma1P1VG7ueSbec4bGcNb13HUCrSDNHd3ytDXG47GnF5GU
API listening on /ip4/127.0.0.1/tcp/5001/http
Gateway (read only) listening on /ip4/127.0.0.1/tcp/8080/http
Web UI available at http://127.0.0.1:5001/webui
Daemon is ready

$ ipfs files mkdir chatterbox
Cannot find module 'form-data'
Require stack:
- /Users/alan/.iim/dists/js-ipfs@0.38.0-rc.5/node_modules/ipfs-http-client/src/add/form-data.js
- /Users/alan/.iim/dists/js-ipfs@0.38.0-rc.5/node_modules/ipfs-http-client/src/add/index.js
- /Users/alan/.iim/dists/js-ipfs@0.38.0-rc.5/node_modules/ipfs-http-client/src/files-regular/index.js
- /Users/alan/.iim/dists/js-ipfs@0.38.0-rc.5/node_modules/ipfs-http-client/src/utils/load-commands.js
- /Users/alan/.iim/dists/js-ipfs@0.38.0-rc.5/node_modules/ipfs-http-client/src/index.js
- /Users/alan/.iim/dists/js-ipfs@0.38.0-rc.5/node_modules/ipfs/src/cli/utils.js
- /Users/alan/.iim/dists/js-ipfs@0.38.0-rc.5/node_modules/ipfs/src/cli/parser.js
- /Users/alan/.iim/dists/js-ipfs@0.38.0-rc.5/node_modules/ipfs/src/cli/bin.js
```

refs https://github.com/ipfs/js-ipfs-http-client/pull/1114
refs https://github.com/ipfs/js-ipfs-http-client/issues/1099

**Comment by achingbrain** [comment]
Probably related: https://github.com/ipfs/js-ipfs-http-client/pull/1114

**Comment by alanshaw** [comment]
Resolved by https://github.com/ipfs/js-ipfs-http-client/pull/1114

**Comment by achingbrain** [comment]
Should be fixed in `0.38.0-rc.6`
